### PR TITLE
Does this title update?

### PR DESCRIPTION
**Pull request by meaghanlewis**

### CODE REVIEW

File: TestWebAppProject/Controllers/HomeController.cs [hunk 1]
```diff
@@ -16,7 +16,6 @@ public IActionResult Index()
         public IActionResult About()
         {
             ViewData["Message"] = "Your application description page.";
-            ViewData["Message"] = "Your application description page.";
 
             return View();
         }
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:21]: cool

Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:21]: cool

Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:21]: cool

Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:21]: comment here

File: TestWebAppProject/Controllers/HomeController.cs [hunk 2]
```diff
@@ -16,7 +16,6 @@ public IActionResult Index()
         public IActionResult About()
         {
             ViewData["Message"] = "Your application description page.";
-            ViewData["Message"] = "Your application description page.";
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:19]: comment here

File: TestWebAppProject/Controllers/HomeController.cs (hunk 1 above)
Comment (meaghanlewis) [reply TestWebAppProject/Controllers/HomeController.cs:21]: reply

Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:21]: here on line 18

File: TestWebAppProject/Controllers/HomeController.cs (hunk 2 above)
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:19]: comment on line 16

Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:19]: line 16?

File: TestWebAppProject/Views/Home/About.cshtml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ﻿@{
-    ViewData["Title"] = "About";
+    ViewData["Title"] = "Aboutttt";
 }
```
Comment (meaghanlewis) [inline TestWebAppProject/Views/Home/About.cshtml:4]: test

File: TestWebAppProject/Views/Home/About.cshtml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ﻿@{
```
Comment (meaghanlewis) [inline TestWebAppProject/Views/Home/About.cshtml:1]: about line 1

File: TestWebAppProject/Controllers/HomeController.cs [hunk 5]
```diff
@@ -16,7 +16,6 @@ public IActionResult Index()
         public IActionResult About()
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:16]: comment on line 16

File: TestWebAppProject/Controllers/HomeController.cs [hunk 6]
```diff
@@ -16,7 +16,6 @@ public IActionResult Index()
...
         {
             ViewData["Message"] = "Your application description page.";
```
Comment (jcansdale) [inline TestWebAppProject/Controllers/HomeController.cs]: viewdata

File: TestWebAppProject/Controllers/HomeController.cs [hunk 7]
```diff
@@ -16,7 +16,6 @@ public IActionResult Index()
         public IActionResult About()
         {
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:17]: commenting with changed local file

File: TestWebAppProject/Controllers/HomeController.cs [hunk 8]
```diff
@@ -16,7 +16,6 @@ public IActionResult Index()
...
             ViewData["Message"] = "Your application description page.";
-            ViewData["Message"] = "Your application description page.";
 
             return View();
```
Comment (grokys) [inline TestWebAppProject/Controllers/HomeController.cs:20]: Does this work now?

File: TestWebAppProject/Controllers/HomeController.cs (hunk 6 above)
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:18]: Hi :smile:

Comment (meaghanlewis) [reply TestWebAppProject/Controllers/HomeController.cs:18]: 👋

File: TestWebAppProject/Views/Home/About.cshtml [hunk 9]
```diff
@@ -1,5 +1,5 @@
...
-    ViewData["Title"] = "About";
+    ViewData["Title"] = "Aboutttt";
```
Comment (ghfvs-tester) [inline TestWebAppProject/Views/Home/About.cshtml:2]: about

File: TestWebAppProject/Views/Home/About.cshtml [hunk 10]
```diff
@@ -1,5 +1,5 @@
...
 }
 
 <p>Use this area to provide additional information.</p>
```
Comment (ghfvs-tester) [inline TestWebAppProject/Views/Home/About.cshtml:5]: <p>Use this area to provide additional information.</p>

Comment (ghfvs-tester) [reply TestWebAppProject/Views/Home/About.cshtml:5]: reply

Comment (ghfvs-tester) [reply TestWebAppProject/Views/Home/About.cshtml:5]: <p>hello</p>

File: TestWebAppProject/Views/Home/About.cshtml (hunk 9 above)
Comment (ghfvs-tester) [inline TestWebAppProject/Views/Home/About.cshtml:2]: new comment from dotcom

**Review by ghfvs-tester — APPROVED** [review]

File: TestWebAppProject/Views/Home/About.cshtml [hunk 11]
```diff
@@ -1,5 +1,5 @@
 ﻿@{
-    ViewData["Title"] = "About";
+    ViewData["Title"] = "Aboutttt";
 }
```
Comment (ghfvs-tester) [inline TestWebAppProject/Views/Home/About.cshtml:3]: <3

Comment (ghfvs-tester) [reply TestWebAppProject/Views/Home/About.cshtml:3]: replying

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]

**Pull request merged**